tests/Integration/DataManagementSystem/Test_DataManager.py

```diff
@@ -36,6 +39,24 @@ def fc():
   return FileCatalog()
 
 
+try:
```

Comment:
@andresailer a small change in the test here. This allows to run the test with other credancials